api/src/Changelog.php

```diff
@@ -38,18 +38,27 @@ public function __construct(
         $contributors = [];
         foreach ($commits as $commit) {
             $commitContributors = CommitParser::extractUsernames($commit->title);
+            $emailUsernameRegex = '/(?<=[0-9]-)([a-zA-Z0-9-_\.]{2,255})(?=@users\.noreply\.drupalcode\.org)/';
```

Comment:
We can define this above the loop since it doesn't change

Reply:
Good call.  Fixed.